### PR TITLE
update cloud proxy image to v1.32.2

### PR DESCRIPTION
**Pull request by cpanato**
**Description of the change**

- update cloud proxy image to v1.32.2

Part of https://github.com/sigstore/public-good-instance/issues/661

 **Existing or Associated Issue(s)**

 **Additional Information**

 

**Checklist**

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). Where applicable, update and bump the versions in any associated umbrella chart
- [ ] Variables are documented in the `values.yaml` and added to the README.md. THe [helm-docs](https://github.com/norwoodj/helm-docs) utility can be used to generate the necessary content. Use `helm-docs --dry-run` to preview the content
- [ ] JSON Schema generated
- [ ] List tests pass for Chart using the [Chart Testing](https://github.com/helm/chart-testing) tool and the `ct lint` command

### CODE REVIEW

File: charts/trillian/README.md [hunk 1]
```diff
@@ -87,7 +87,7 @@ The following table lists the configurable parameters of the trilian chart and t
 | mysql.enabled | bool | `true` |  |
 | mysql.gcp.cloudsql.registry | string | `"gcr.io"` |  |
 | mysql.gcp.cloudsql.repository | string | `"cloudsql-docker/gce-proxy"` |  |
-| mysql.gcp.cloudsql.version | string | `"1.28.1"` |  |
+| mysql.gcp.cloudsql.version | string | `"sha256:7755f632c090289b8e0fa67218fc9d8b9ab7e0758ea58c48786617a79ff715dc"` |  |
```
Comment (k4leung4) [inline charts/trillian/README.md:90]: add version to description column?

**Review by k4leung4 — DISMISSED** [review]

File: charts/trillian/README.md (hunk 1 above)
Comment (cpanato) [reply charts/trillian/README.md:90]: thanks for the catch!

Comment (cpanato) [reply charts/trillian/README.md]: done, thanks!

**Review by hectorj2f — APPROVED** [review]

**Review by k4leung4 — DISMISSED** [review]

**Comment by cpanato** [comment]
already updated, closing this

**Pull request closed without merging**